pyqtorch/__init__.py

```diff
@@ -100,6 +101,8 @@
 )
 
 __all__ = [
+    "ConcretizedCallable",
+    "Embedding",
```

Comment:
hm but in platforms we would access them through that. i would keep it like that in case you guys decide to move these things to other modules. its very annoying to have to adjust imports always